assets/js/modules/analytics/datastore/profiles.js

```diff
@@ -162,11 +161,11 @@ export const actions = {
 		};
 	},
 
-	receiveProfiles( profiles ) {
+	receiveProfiles( profiles, { propertyID } ) {
 		invariant( Array.isArray( profiles ), 'profiles must be an array.' );
```

Comment:
Can we add an `invariant` call here to ensure `propertyID` is not `undefined`? This ensures that we always have to pass it, which we should (see other comments on empty objects in tests above).

Reply:
Fixed